### PR TITLE
New Awesome Window Manager Cheat Sheet

### PR DESCRIPTION
**Pull request by kennyballou**
This provides a new Awesome WM cheat sheet/reference page for default key
bindings, mouse bindings, and reference material for configuration.

[IA Page][1]

[1]: https://duck.co/ia/view/awesome_window_manager_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@kennyballou Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=awesome window manager cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/awesome_window_manager_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/awesome-window-manager.json [hunk 1]
```diff
@@ -0,0 +1,248 @@
+{
+    "id": "awesome_window_manager_cheat_sheet",
+    "name": "Awesome Window Manager Cheat Sheet",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/awesome-window-manager.json]: @kennyballou Lets remove "cheat sheet" from name

**Comment by kennyballou** [comment]
@jophab Would you like me to squash the commits and resubmit/force push?

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=awesome window manager cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/awesome_window_manager_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=awesome window manager cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/awesome_window_manager_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=awesome window manager cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/awesome_window_manager_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=awesome window manager cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/awesome_window_manager_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/awesome-window-manager.json [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+        "Mouse",
+        "Navigation",
+        "Layout Modification",
+        "Important Files",
+        "Links and Information"
+    ],
+    "sections": {
+        "Window Manager Control": [
+            {
+                "key": "Mod4 + Ctrl + r",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/awesome-window-manager.json]: For all `key` values that contain multiple keyboard keys, you need to wrap each key in `[]` and remove the `+`.

E.g. `"[Mod4] [Ctrl] [r]"`

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=awesome window manager cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/awesome_window_manager_cheat_sheet_mobile.png)

**Comment by kennyballou** [comment]
@moollaza while I'm changing this, is there anything special I need to do for the numerical options, e.g., `[1-9]`?

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=awesome window manager cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/awesome_window_manager_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=awesome window manager cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/awesome_window_manager_cheat_sheet_mobile.png)

**Comment by moollaza** [comment]
@kennyballou if you want to show literal `[]` you'll need to escape them: `{\\[1-9\\]}`

Please also switch this from the `reference` template to the `keyboard` template :+1:

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=awesome window manager cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/awesome_window_manager_cheat_sheet_mobile.png)

**Comment by kennyballou** [comment]
@moollaza After referencing the Apple OSX cheat sheet, I feel the output of the current format should convey the proper keys and usage. So let me know, if anything, are the next steps.

File: share/goodie/cheat_sheets/json/awesome-window-manager.json [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+            {
+                "key": "User Configuration",
+                "val": "$XDG_CONFIG_HOME/awesome/rc.lua"
+            },
+            {
+                "key": "System/Default Confiruation",
+                "val": "/etc/xdg/awesome/rc.lua"
+            }
+        ],
+        "Links and Information": [
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/awesome-window-manager.json]: @kennyballou let's remove this section -- the info can be found at the Source and it's also breaking the UI (not your fault)

awesome_window_manager_cheat_sheet_at_duckduckgo

**Comment by kennyballou** [comment]
@moollaza Just the links and information section or the quoted block? Looks like the links and information page is all we need to delete, yes?

**Comment by kennyballou** [comment]
Looks like (on [beta][1]) the important files section also breaks the UI if forced to two columns. I will remove both sections.

[1]: https://beta.duckduckgo.com/?q=awesome+window+manager+cheat+sheet&ia=cheatsheet&iax=1

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=awesome window manager cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/awesome_window_manager_cheat_sheet_mobile.png)

**Comment by kennyballou** [comment]
Is this ready to merge? Should I squash and resubmit?

**Comment by moollaza** [comment]
@kennyballou Thanks, and sorry for the delay!  This LGTM :+1:

**Comment by tagawa** [comment]
@kennyballou, thanks again for creating the Awesome WM cheat sheet! You’ve done a really good job here and we'd like you to help make DuckDuckGo even better for yourself and others interested in desktop customization.

When you're ready to move on, please:

- Create a new Instant Answer for searches you do related to desktop customization

OR

- Test live desktop customization Instant Answers (or those in development) for bugs and suggest improvements

Detailed instructions explaining how you can help are available at the top of this [page](https://github.com/duckduckgo/duckduckgo/wiki/Creating-or-Improving-Instant-Answers).

Thanks again!

**Pull request merged**